schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name varchar(100),
+    date_of_birth date,
+    escape_attempts INT,
+    neutered boolean,
+    weight_kg decimal
 );
```

Comment:
- [OPTIONAL] I suggest to mark the id column as primary key 💯
```sql
   PRIMARY KEY (id)
```

- [OPTIONAL] I suggest to mark the id column not null 💯
```sql
  id int NOT NULL IDENTITY(1,1)
```